src/server/metrics.rs

```diff
@@ -50,21 +52,25 @@ make_static_metric! {
         unsafe_destroy_range,
     }
 
-    pub struct GrpcMsgHistogramVec: Histogram {
+    pub struct GrpcMsgHistogramVecLocal: LocalHistogram {
         "type" => GrpcTypeKind,
     }
 
-    pub struct GrpcMsgFailCounterVec: IntCounter {
+    pub struct GrpcMsgHistogramVecGlobal: Histogram {
         "type" => GrpcTypeKind,
     }
 
-    pub struct GcCommandCounterVec: IntCounter {
+    pub struct GrpcMsgFailCounterVec: LocalIntCounter {
+        "type" => GrpcTypeKind,
+    }
+
+    pub struct GcCommandCounterVec: LocalIntCounter {
         "type" => GcCommandKind,
     }
 }
 
 lazy_static! {
-    pub static ref SEND_SNAP_HISTOGRAM: Histogram = register_histogram!(
+    pub static ref SEND_SNAP_HISTOGRAM_GLOBAL: Histogram = register_histogram!(
```

Comment:
I prefer to remove these `pub` visibility for global metrics as more as possible, so that it can be intuitively seen that they are not used globally.

Reply:
Most `pub` prefix are removed,but I kept `pub` of `GRPC_MSG_HISTOGRAM_GLOBAL`, because `src/server/service/batch.rs:323` `ReqBatcher` need to read it to decide batch rate or something

Reply:
@hicqu PTAL if the localization of the metric observing in elsewhere will affect this place.